frontend/src/app/draft/page.tsx

```diff
@@ -0,0 +1,84 @@
+"use client";
+
+import Image from "next/image";
+import Link from "next/link";
+import { LinkButton } from "@ui/LinkButton";
+
+const Draft = () => {
+  return (
+    <section className="w-full py-10 mt-5 bg-ellipse bg-no-repeat bg-contain bg-right">
+      <div className="w-4/6 mx-auto">
+        <div className="relative">
+          <Image
+            src="/assets/draft.png"
+            alt="book-title"
+            width={200}
+            height={3}
+          />
+          <div className="absolute top-1/2 left-20 transform -translate-x-1/2 -translate-y-1/2 bg-opacity-70 p-2 whitespace-nowrap">
+            <h1 className="mt-0 font-normal text-3xl md:text-4xl text-green font-quicksand tracking-normal md:tracking-widest leading-none md:leading-normal lg:leading-loose">
+              Draft{" "}
+            </h1>
+          </div>
+        </div>
+      </div>
+
+      <div className="w-4/6 my-10 mx-auto">
+        <h2 className="text-xl">
+          <span className="font-semibold">Title :</span> Our Family{" "}
+        </h2>
+        <h2 className="text-xl mt-3">
+          <span className="font-semibold">Story :</span> The story that was
+          written in the create page{" "}
+        </h2>
+        <h2 className="text-xl mt-3 font-semibold">
+          Back to{" "}
+          <Link href="/create-story" className="text-pink">
+            Create your{" "}
+            <span className="font-pacifico font-normal">Story!</span>
+          </Link>
+        </h2>
+      </div>
+
+      <div className="h-[300px] bg-flower bg-no-repeat bg-contain bg-[center_right_20rem]"></div>
```

Comment:
If you follow the Figma design, UI for the background flower image is not correct! Would you please make specific changes to that?  Let me know if you need more details!

Reply:
Fix background flower image position  as per UI